```
20:44:47 
/Users/buildnode/jenkins/workspace/swift-lldb-PR-osx/lldb/source/Symbol/SwiftASTContext.cpp:6071:45: error: no member named 'Get' in 'lldb_private::SwiftLanguageRuntime'
20:44:47   if (auto *runtime = SwiftLanguageRuntime::Get(*exe_scope->CalculateProcess()))
20:44:47                       ~~~~~~~~~~~~~~~~~~~~~~^
20:44:47 1 error generated.
```

Looks like I need to adapt this for 5.1, but the general idea is there.

--- a/source/Symbol/SwiftASTContext.cpp
+++ b/source/Symbol/SwiftASTContext.cpp
@@ -6068,7 +6068,7 @@ SwiftASTContext::GetByteStride(lldb::opaque_compiler_type_t type,
 
   if (!exe_scope)
     return {};
-  if (auto *runtime = SwiftLanguageRuntime::Get(*exe_scope->CalculateProcess()))
+  if (auto *runtime = exe_scope->CalculateProcess()->GetSwiftLanguageRuntime())
     return runtime->GetByteStride({this, type});
   return {};
 }

